topics/researchcloud.qmd

```diff
@@ -0,0 +1,47 @@
+
+SURF Research Cloud is hosted at the SURF datacenter in Amsterdam. 
+
+## What can it be used for?
+SURF Research Cloud enables you to run 1 or more servers with applications that can be accessible over the internet.
+
+- Hosting web applications, these can be connected to SRAM for secure authentication. You can easily start more servers to increase availability and preformance.
+
+- Running a desktop environment with analysis tools, use your favorite desktop tools on an environment with more performance. Unlike [SciCloud][./scicloud.qmd] GPU resources are available. You can easily connect environment to your data on Research Drive or Yoda.
+
+- Hosting a highly secure research environment where sensitive data cannot leave the VRE: [SURF SANE](https://www.surf.nl/en/themes/research-infrastructure/sane-secure-environment-for-analysing-sensitive-data).
+
+## How to request access
+All VU researchers can get access to SURF Research Cloud. To start building an environment in Research Cloud you need 2 things:
+
+**1. An SRAM Collaboration connected to Research Cloud**
+
+- First apply for a new SRAM Collaboration (CO): log in to [SRAM](https://sram.surf.nl) and click the "Request collaboration" button. 
+
+- The SRAM admins at IT for Research (ITvO) will contact you to discuss your specific needs and help you to get started.
+
+- Once the Collaboration request is approved you can connect the Research Cloud Application to the new collaboration.
+
+- Invite the colleagues that need to work in your VRE to your new CO.
+
+**2. A Wallet with credits**
+
+There are 2 ways to obtain credits:
+
+- Apply for a _NWO Small Compute Grant_ by following the links on [this page](https://www.surf.nl/en/small-compute-applications-nwo). In the request form select "SURF Research Cloud - HPC Cloud".
+
+- If for whatever reason your application for this grant is denied, you could also claim credits from the VU [RCCS](https://www.surf.nl/research-capacity-computing-service-rccs-directe-toegang-tot-onze-rekensystemen-en-expertise) contract. Please contact [IT for Research](mailto:itvo.it@vu.nl) for details on how to obtain these credits.
+
+Once the request is granted you will have access to a _wallet_ in Research Cloud and can start to build an environment.
+
+## Getting started 
+Documentation for Research Cloud can be found on the [SURF User Knowledge Base](https://servicedesk.surf.nl/wiki/display/WIKI/SURF+Research+Cloud).
+
+[IT for Research](mailto:itvo.it@vu.nl) can assist you in setting up a new Research Cloud environment.
+
+## Contact
+Wondering if SURF Research Cloud fits your research needs? Please contact [IT for Research](mailto:itvo.it@vu.nl)
```

Comment:
No examples yet. 

There are a few researchers at the VU using Research Cloud but they have only been in contact with SURF and we do not know exactly what they are using it for. 

Since the start of the year we are connected to SRAM and researchers will need our approval first, but so far we have had no requests...

Reply:
My proposal is to leave this for now. As said it is new and I think elaborating on the RC Catalogue and Ansible scripts is a bit too much. I will revisit once we have seen some more concrete use-cases.